app/src/main/java/com/globant/example/mentorapp/home/presentation/view/fragment/ListUsersViewFragment.java

```diff
@@ -0,0 +1,144 @@
+import android.arch.lifecycle.ViewModelProviders;
+import android.os.Bundle;
+import android.support.annotation.Nullable;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.support.v7.widget.StaggeredGridLayoutManager;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.globant.example.mentorapp.MentorApplication;
+import com.globant.example.mentorapp.R;
+import com.globant.example.mentorapp.home.presentation.model.ListUsersViewModel;
+import com.globant.example.mentorapp.home.presentation.model.ModelUserEntity;
+import com.globant.example.mentorapp.home.presentation.model.SharedUserViewModel;
+import com.globant.example.mentorapp.home.presentation.presenter.ListUsersPresenterImpl;
+import com.globant.example.mentorapp.home.presentation.view.adapter.ListUsersAdapter;
+import com.globant.example.mentorapp.mvp.base.BaseActivity;
+import com.globant.example.mentorapp.mvp.base.BaseModel;
+import com.globant.example.mentorapp.mvp.base.BaseView;
+
+import java.util.List;
+
+import javax.inject.Inject;
+
+/**
+ * A placeholder fragment containing a list of Users with his name.
+ */
+public class ListUsersViewFragment extends LifecycleFragment implements BaseView {
+
+    public static final String LIST_TAG = "listUserFragment";
+    @Inject
+    public ListUsersPresenterImpl presenter;
+    protected StaggeredGridLayoutManager staggeredGridLayoutManager;
+    private SharedUserViewModel model;
+    private ListUsersAdapter listUsersAdapter;
+    private RecyclerView listUsersRecyclerView;
+    private BaseActivity parent;
+
+    @Override
+    public void onCreate(@Nullable Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        ((MentorApplication) getActivity().getApplicationContext()).getApplicationComponent().inject(this);
+    }
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        View view = inflater.inflate(R.layout.fragment_home_screen, container, false);
+        model = ViewModelProviders.of(this).get(SharedUserViewModel.class);
+        parent = (BaseActivity) getActivity();
+        staggeredGridLayoutManager = new StaggeredGridLayoutManager(
+                getResources().getInteger(R.integer.number_of_users_columns), LinearLayoutManager.VERTICAL);
+        if (model.getUsers() != null || model.getUsers().getValue() != null) {
+            listUsersAdapter = new ListUsersAdapter(model.getUsers().getValue(), getContext());
+            parent.hideProgress();
+        }
+        listUsersRecyclerView = (RecyclerView) view.findViewById(R.id.list_item_container);
+        defineRecyclerView(listUsersRecyclerView, listUsersAdapter);
+        return view;
+    }
+
+    @Override
+    public void render(BaseModel baseModel) {
+        ListUsersViewModel usersModel = (ListUsersViewModel) baseModel;
```

Comment:
You can declare the parameter as ListUsersViewModel since it extends from BaseModel, so you avoid casting objects